framework/src/araui/core/ui_core.c

```diff
@@ -306,6 +314,22 @@ static void _ui_call_tween_finished_cb(void *userdata)
 	}
 }
 
+static void _ui_call_anim_finished_cb(void *userdata)
+{
+	ui_widget_body_t *body;
+
+	if (!userdata) {
+		UI_LOGE("error: Invalid Parameter!\n");
+		return;
+	}
+
+	body = (ui_widget_body_t *)userdata;
+	if (body && body->anim_finished_cb) {
+		body->anim_finished_cb((ui_widget_t)body);
+	}
+	body->anim_finished_cb = UI_NULL;
```

Comment:
If body is null, this sentence occurs an error

Reply:
OK I will fix next PR